src/viztracer/modules/snaptrace.c

```diff
@@ -1190,6 +1205,14 @@ snaptrace_config(TracerObject* self, PyObject* args, PyObject* kw)
         strcpy(self->lib_file_path, kw_lib_file_path);
     }
 
+    if (kw_process_name && kw_process_name != Py_None) {
+        if (self->process_name) {
+            Py_DECREF(self->process_name);
+        }
+        self->process_name = kw_process_name;
+        Py_INCREF(self->process_name);
```

Comment:
```suggestion
        Py_INCREF(kw_process_name);
        Py_XSETREF(self->process_name, kw_process_name);  
```

Reply:
Maybe also add a unicode check here - to make sure toe process name given is a string, otherwise raise an exception.